addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -565,13 +567,15 @@
                     <field name="total_amount"/>
                     <field name="accounting_date"/>
                     <field name="state"/>
+                    <field name="currency_id"/>
                     <templates>
                         <t t-name="kanban-box">
                             <div t-attf-class="oe_kanban_card oe_kanban_global_click">
                                 <div class="row">
                                     <div class="col-12">
                                         <strong class="o_kanban_record_title"><span><t t-esc="record.name.value"/></span></strong>
                                         <strong class="o_kanban_record_subtitle float-right"><span class="text-right"><t t-esc="record.total_amount.value"/></span></strong>
+                                        <strong class="o_kanban_record_subtitle float-right mr4"><span class="text-right"><t t-esc="record.currency_id.value"/></span></strong>
```

Comment:
can you use the 'monetary' widget on the amount field ? in order to have the currency symbol on the kanban card ? Thanks

Reply:
@jem-odoo done.